lerna.json

```diff
@@ -1,7 +1,9 @@
 {
   "lerna": "2.8.0",
   "packages": [
-    "handel"
+    "handel",
+    "extension-api",
+    "official-extensions"
```

Comment:
This looks like a placeholder for something you had in mind. Are you thinking to split out what's currently in the services/ directory in Handel to be modules inside this directory, or were you thinking this would be for something else?

Reply:
Huh. I had meant to remove that; I was just seeing how Lerna handled paths that don't exist (spoiler alert: It just ignores them!).

I had thought of adding 'official' extensions at one point to add some interesting services, but then I realized that we should probably just add anything like that into the core library 🙃